app/src/main/res/layout/screen_overdue_search.xml

```diff
@@ -106,17 +106,48 @@
     app:layout_constraintStart_toStartOf="parent"
     app:layout_constraintTop_toBottomOf="@id/overdue_search_app_bar" />
 
+  <LinearLayout
+    android:id="@+id/select_all_layout"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginStart="@dimen/spacing_20"
+    android:layout_marginTop="@dimen/spacing_24"
+    android:orientation="horizontal"
+    android:visibility="gone"
+    app:layout_constraintTop_toBottomOf="@+id/overdue_search_app_bar">
+
+    <com.google.android.material.checkbox.MaterialCheckBox
+      android:id="@+id/select_all_checkbox"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:minWidth="@dimen/spacing_24"
+      android:minHeight="@dimen/spacing_24"
+      android:textColor="?attr/colorPrimary" />
```

Comment:
Irrespective of the state, the checkbox color is supposed to be primary. Let's update it.

Reply:
The color updated here is for the textview, not for the checkbox, PTAL

Reply:
I am talking about the checkbox color, not the text color. But we might end up removing the checkbox altogether. 

https://simpledotorg.slack.com/archives/CFK2PHJ1L/p1658145294726409

![Screenshot 2022-07-18 at 5.41.46 PM.png](https://graphite-user-uploaded-assets.s3.amazonaws.com/KqrtAphEKX3ow2MZW1lw/4c3c2a1a-d8fe-4116-8f88-21b8a0dc748d/Screenshot%202022-07-18%20at%205.41.46%20PM.png)

Reply:
If we end up keeping checkbox. We need to update the text and check box color